docs/codacy-api/using-the-codacy-api.md

```diff
@@ -173,3 +173,12 @@ If you continue requesting more pages the endpoint will eventually return a `pag
   }
 }
 ```
+
+## Request rate limit {: id="rate-limit"}
+
+Codacy limits the number of requests that you can perform to the Codacy API to help us provide a reliable service:
+
+-   The limit is **2500 requests per 5 minutes and per source IP address**
```

Comment:
is it only per IP address ignoring the account or token used? 😮 
what happens with API calls sent from GitHub and other CI infrastructure?

Reply:
I got this information from this @h314to's [reply on Slack](https://codacy.slack.com/archives/CAL88HZL6/p1650552924260839?thread_ts=1650552195.204769&cid=CAL88HZL6). Perhaps he can help clarify better.